<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRulePatternInputWidget</class>
 <widget class="QWidget" name="SPRulePatternInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Node Type Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="typeNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="TextEditor" name="plainTextEdit">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ElementListWidget" name="elementListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>src/gui/TextEditor.h</header>
  </customwidget>
  <customwidget>
   <class>ElementListWidget</class>
   <extends>QListWidget</extends>
   <header>src/gui/ElementListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
